.github/workflows/test.yaml

```diff
@@ -18,8 +32,6 @@ jobs:
           path: "~/.cache/bazel"
           key: bazel
 
-      - uses: pre-commit/action@v2.0.3
-
```

Comment:
Consider moving the `./bin/ensure-unchanged bazel run //internal/parser:update` to the linux only job.

Reply:
Yes, that is actually what I wanted to say. We could run the linux tests in a separate job and then depend on that job and only run `bazel test //...` on windows. This way we can mark the Windows job as non-blocking the merge for some time and we always get the linux test run before hand.